app/helpers/sessions_helper.rb

```diff
@@ -3,15 +3,30 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if user_id = session[:user_id]
+      @current_user ||= User.find_by_id user_id
```

Comment:
It is not recommended to use dynamic finder(find_by_id) now. You can use find_by here.